mets.md

```diff
@@ -28,6 +56,40 @@ $> exiftool output.tif |grep 'X Resolution'
 "150"
 ```
 
+## Images and coordinates
+
+Coordinates are always absolute, i.e. relative to extent defined in the `imageWidth`/`imageHeight` attribute of the nearest `<pc:Page>`.
+
+When a processor wants to access the image of a layout element like a TextRegion or TextLine, the algorithm should be:
+
+- If the element in question has an attribute `imageFilename`, resolve this value
+- If the element has a `<pc:Coords>` subelement, resolve by passing the attribute `imageFilename` of the nearest `<pc:Page>` and the `points` attribute of the `<pc:Coords>` element
+
+
+# Font information
+
+The documentation of these features is also stored in the **METS file**. In
+this case, this information is extracted from the page file. 
+
+The information is documented in the `<dmSec>` area.
+
+```xml
+<dmSec ID="dmd001">
+  <mdWrap MIMETYPE="text/XML" MDTYPE="PAGEXML" LABEL="PAGE XML">
+    <xmlData>
+      <page:TextRegion id="r_1_1" custom="textStyle {fontFamily:Arial:Times:Courier; }">
```

Comment:
Is colon `:` really the best separator? In CSS, font-families are separated by comma, in double quotes if containing a space:

```css
{ fontFamily: Arial, "Times New Roman", Courier; }
```

IIUC it cannot be avoided that data consumers will need to parse the `custom` attribute. Is this format defined somewhere (by Transkribus I suppose)? Could this be replaced with something easy to parse like JSON?